nixos/modules/services/x11/display-managers/default.nix

```diff
@@ -33,7 +33,7 @@ let
       #! ${pkgs.bash}/bin/bash
 
       # Handle being called by SDDM.
-      if test "''${1:0:1}" = / ; then eval exec $1 $2 ; fi
+      if test "''${1:0:1}" = / ; then eval exec "$1" "$2" ; fi
```

Comment:
Without being able to reasonably test the change, I was trying to limit it to things I was 100% sure would work.